Python/Product/TestAdapter/TestContainerDiscovererWorkspace.cs

```diff
@@ -288,5 +290,31 @@ class TestContainerDiscovererWorkspace : ITestContainerDiscoverer, IDisposable {
             }
             NotifyContainerChanged();
         }
+
+        private IEnumerable<string> FilterInvalidFiles(ProjectInfo projectInfo) {
+            List<string> validFiles = Directory.EnumerateFiles(projectInfo.ProjectHome, "*.py", SearchOption.TopDirectoryOnly).ToList();
+            List<InterpreterConfiguration> workspaceInterpreterFactories = _interpreterRegistryService.Configurations
+                .Where(x => PathUtils.IsSubpathOf(projectInfo.ProjectHome, x.InterpreterPath))
+                .ToList();
+
+            foreach (var directory in Directory.EnumerateDirectories(projectInfo.ProjectHome, "*", SearchOption.TopDirectoryOnly)) {
```

Comment:
Those are the defaults, you can just do `Directory.EnumerateDirectories(projectInfo.ProjectHome)`